From 0616c7372e9738c88fe6bc3678468f6e1a2e2fd0 Mon Sep 17 00:00:00 2001
From: Jianing Ren <jianing.ren@rock-chips.com>
Date: Fri, 1 Nov 2019 18:02:58 +0800
Subject: [PATCH] arm64: dts: rk3399-evb-lpddr4: fix the vbus-5v-gpios active
 level

The power of fusb302 is controlled by GPIO1_C2 which should be set as
ACTIVE_HIGH and it was set incorrectly in this dtsi file leading to cannot
send notifie.

Change-Id: Ibc045d266e5bc9718343e07acda8488b0d747aba
Signed-off-by: Jianing Ren <jianing.ren@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
index 07da6487b545..af7fda60f756 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind.dtsi
@@ -796,7 +796,7 @@
 		pinctrl-names = "default";
 		pinctrl-0 = <&fusb0_int>;
 		int-n-gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
-		vbus-5v-gpios = <&gpio1 18 GPIO_ACTIVE_LOW>;
+		vbus-5v-gpios = <&gpio1 18 GPIO_ACTIVE_HIGH>;
 		status = "okay";
 	};
 
-- 
2.35.3

